tests/plugins/inventory/test_netbox.py

```diff
@@ -38,10 +49,11 @@ def transform_function(host):
 
 class Test(object):
     def test_inventory(self, requests_mock):
-        inv = get_inv(requests_mock, "2.3.5", False)
-        with open("{}/{}/expected.json".format(BASE_PATH, "2.3.5"), "r") as f:
-            expected = json.load(f)
-        assert expected == Inventory.serialize(inv).dict()
+        for version in ["2.3.5", "2.8.9"]:
```

Comment:
I think it would be better to add a class variable `versions` which is a list of versions.
Iterate over that class variable in each test.

Reply:
Added